dashboard/src/modules/components/AuthComponent/common-components.jsx

```diff
@@ -117,7 +117,12 @@ export const AuthForm = () => {
       <CardFooter>
         <div className="log-in-alternate">Or log in with...</div>
         <div className="alternate-btn-wrapper">
-          <Button variant="secondary">Red Hat SSO</Button>
+          <Button
+            variant="primary"
+            onClick={() => {dispatch(authenticationRequest())}}
+          >
+            Red Hat SSO
+          </Button>
           <Button variant="secondary">GitHub</Button>
           <Button variant="secondary">Gmail</Button>
```

Comment:
Wasn't the idea that we'd redirect to the Keycloak broker and let it display the menu of SSO options? Are we really skipping a layer here, or redirecting directly to Red Hat?

Reply:
Yeah, I didn't change the button name but I should update it its not a Red Hat SSO button anymore.